app/src/question/index.js

```diff
@@ -10,6 +10,9 @@ var voteActions = require('vote/actions');
 
 var voteDuration = timeUtil.second * 30;
 
+var api = require('lib/api');
+var councilToken = api.getCouncilToken();
```

Comment:
Don't call this function up here. Call it when you need it. And then tell me why that is the case.